common/src/main/java/software/tnb/common/openshift/OpenshiftClient.java

```diff
@@ -400,4 +402,37 @@ public String getIntegrationContainer(Pod pod) {
         }
         return container;
     }
+
+    public SecurityContextConstraints createSecurityContext(String sccName, String copyFromScc, String... defaultCapabilities) {
+        SecurityContextConstraints scc = OpenshiftClient.get().securityContextConstraints().withName(sccName).get();
+        if (scc == null) {
+            SecurityContextConstraints existingScc = OpenshiftClient.get().securityContextConstraints().withName(copyFromScc).get();
+            scc = OpenshiftClient.get().securityContextConstraints().create(
+                new SecurityContextConstraintsBuilder(existingScc)
+                    .withNewMetadata() // new metadata to override the existing annotations
+                    .withName(sccName)
+                    .endMetadata()
+                    .addToDefaultAddCapabilities(defaultCapabilities)
+                    .build());
+        }
+        return scc;
+    }
+
+    public void addUsersToSecurityContext(SecurityContextConstraints scc, String... users) {
+        for (String user : users) {
+            if (!scc.getUsers().contains(user)) {
+                scc.getUsers().add(user);
+            }
+        }
+        OpenshiftClient.get().securityContextConstraints().withName(scc.getMetadata().getName()).patch(scc);
+    }
+
+    public void addGroupsToSecurityContext(SecurityContextConstraints scc, String... groups) {
+        for (String group : groups) {
+            if (!scc.getGroups().contains(group)) {
+                scc.getGroups().add(group);
+            }
+        }
+        OpenshiftClient.get().securityContextConstraints().withName(scc.getMetadata().getName()).patch(scc);
+    }
```

Comment:
perhaps we could also add something like `createServiceAccountRef(name)` while we're at it? as there are multiple instances of `"system:serviceaccount:" + OpenshiftConfiguration.openshiftNamespace() + "somename"`